Podfile

```diff
@@ -16,6 +16,7 @@ def ios_pods
     pod 'PinpointKit', :git => 'https://github.com/Lickability/PinpointKit', :commit => 'dd5731c'
     pod 'DZNEmptyDataSet', '1.8.1'
     pod 'Hero'
+    pod 'SimpleRoundedButton'
```

Comment:
Idea is to have one button toolclass for Schul-Cloud, this pod is from Carl and brings some extra convience